styles.css

```diff
@@ -1,26 +1,66 @@
 /*credit to w3schools.com for tutorials on web design*/
 #background {
     height: 100%;
-    background-image: url(https://utemplates.net/wp-content/uploads/2017/07/White-Textured-Background.jpg);
+    background-image: url(https://utemplates.net/wp-content/uploads/2017/07/White-Textured-Background.jpg);    ;
+}
+
+h1, h2,h3,h4, h5, h6, p,form,div,table{
+  font-family: 'Ubuntu';
+  color: maroon;
 }
 
 #title {
     color: maroon;
     text-align: center;
     font-family: 'Ubuntu';
     border-width: 5px;
-    margin-top: 2%;
+    margin-right: 10%;
+    display: block;
 }
 
-#navBtn
+#buttons
 {
     color: gold;
     background-color: maroon;
-    font-family: 'Ubuntu';
-    float:left;
-    display:block;
-    margin-left:auto;
-    margin-right:auto;
+    border-color: black;
+    border-radius: 12px;
+    font-size:16px;
+    margin: 4px 2px;
+}
+
+.buttons
+{
+  color: gold;
+  background-color: maroon;
+  border-color: black;
+  border-radius: 12px;
+  font-size:16px;
+  margin: 4px 2px;
+}
```

Comment:
Apply this to all buttons (or rather, all buttons with `id="buttons")?

This may be desirable right now, or even with buttons in general, but you should use the class setup
```CSS
.r3-button
{
  color: gold;
  background-color: maroon;
  border-color: black;
  border-radius: 12px;
  font-size:16px;
  margin: 4px 2px;
}
```
and then in the html
```html
<button class="r3-button">blah</button>
```

(I'm using r3 for r^3, like it's fine to do this for what you want the "standard" button on the site to be,